services/editor/src/components/App.js

```diff
@@ -1,68 +1,48 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
-import { Route } from 'react-router';
 import { Observable } from 'rxjs/Rx';
 import { setObservableConfig, compose } from 'recompose';
-import classNames from 'classnames';
+import { connect } from 'react-redux';
+
+import registerServiceWorker from '../registerServiceWorker';
 import withLoading from '../hoc/with-loading';
+import { isAuthenticated } from '../services/auth-service';
 import { refreshSchema } from '../services/context-service';
 import * as TypesService from '../services/types-service';
-import Alerts from './alerts/Alerts';
-import Notifications from './alerts/Notifications';
+import { redirectToLogin } from '../store/ducks/login';
+import AppHeader from './AppHeader';
+import AppPage from './AppPage';
+import GoogleTagManager from './googleTagManager';
+
 import { withTypesService } from './common/Input/TypedInput';
-import logoSrc from './resources/logo.svg';
 import '../styles/core/fonts/fonts.css';
 import './App.css';
 
 setObservableConfig({
   fromESObservable: Observable.from,
 });
 
-const ListItemLink = ({ to, ...rest }) => (
-  <Route
-    path={to}
-    children={({ match }) => (
-      <li>
-        <Link
-          className={classNames('menu-item', {
-            'selected-location-path': match,
-          })}
-          to={to}
-          {...rest}
-        />
-      </li>
-    )}
-  />
+const App = ({ children }) => (
+  <div className={'app'}>
+    <GoogleTagManager />
+    <AppHeader />
+    <AppPage children={children} />
+  </div>
 );
 
-export default compose(
-  withLoading(() => null, () => Promise.all([TypesService.refreshTypes(), refreshSchema()])),
+const preload = async ({ redirectToLogin }) => {
```

Comment:
If routing is fixed and app component is loading only for not "/login" paths, we can assume we are authenticated because of server side redirect.